pkgs/by-name/op/ophcrack/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, libsForQt5
+, fetchurl
+, fetchpatch
+, autoreconfHook
+, pkg-config
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ophcrack";
+  version = "3.8.0";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/ophcrack/files/ophcrack/3.8.0/ophcrack-3.8.0.tar.bz2";
+    hash = "sha256-BIpt9XmDo6WjGsfE7BLfFqpJ5lKilnbZPU75WdUK7uA=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    libsForQt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    pkg-config
+    openssl
+    libsForQt5.qtcharts
+  ];
+
+  patches = [
+    (fetchpatch {
+      url = "https://salsa.debian.org/pkg-security-team/ophcrack/-/raw/c60118b40802e1162dcebfe5f881cf973b2334d3/debian/patches/fix_spelling_error.diff";
+      hash = "sha256-Fc044hTU4Mtdym+HukGAwGzaLm7aVzV9KpvHvFUG2Sc=";
+    })
+    (fetchpatch {
+      url = "https://salsa.debian.org/pkg-security-team/ophcrack/-/raw/e19d993a7dbf131d13128366e2aac270a685befc/debian/patches/qmake_crossbuild.diff";
+      hash = "sha256-sOKXOBpAYGLacU6IxjRzy3HCnGm4DFowDL2qP+DzG8M=";
+    })
+  ];
+
+  dontWrapQtApps = true;
+
+  configureFlags = [
+    "--with-libssl=yes --enable-gui --enable-debug"
+  ];
+
+  buildPhase = ''
+    make
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/ophcrack \
+      --prefix PATH : "${libsForQt5.qtbase}/bin"
```

Comment:
Is this really needed? Have you tried adding qtbase to `buildInputs`?

Reply:
Yes i did try having qtbase in buildInputs without postInstall it results to,
"qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Aborted (core dumped)"

Reply:
i think this postInstall not needed anymore